mods/Deco@Pokerleven/meta.json

```
@@ -0,0 +1,15 @@
+{
+  "title": "Pokerleven",
+  "requires-steamodded": true,
+  "requires-talisman": true,
+  "categories": [
+    "Content",
+    "Joker"
+  ],
+  "author": "Deco",
+  "folderName": "Pokerleven",
+  "repo": "https://github.com/DecoXFE/PokerLeven",
+  "downloadURL": "https://github.com/DecoXFE/PokerLeven/releases/latest",
```

this url is invalid because it doesn't point directly at a zip file, given your current repo's state this is the only valid release-based auto-updating url
```suggestion
  "downloadURL": "https://github.com/DecoXFE/PokerLeven/archive/refs/tags/v0.0.3b-alpha.zip",
```